### PR TITLE
Additional times

### PR DESCRIPTION
**Pull request by BKorenek**
This is a PR to add additional time formats for our testing suites to utilize.

### CODE REVIEW

File: src/Medology/Behat/StoreContext.php [hunk 1]
```diff
@@ -32,17 +32,20 @@ class StoreContext extends Store implements Context
     protected static $dateFormat = DateTime::ISO8601;
 
     protected static $format_map = [
-        'a MySQL date'                    => 'Y-m-d',
-        'a MySQL date and time'           => 'Y-m-d H:i:s',
-        'a US date'                       => 'm/d/Y',
-        'a US date and time'              => 'm/d/Y H:i:s',
-        'a US time'                       => 'h:i',
-        'a US military time'              => 'H:i',
-        'a US time with seconds'          => 'h:i:s',
```
Comment (aserv92) [inline src/Medology/Behat/StoreContext.php:41]: I'm don't think we should remove this unless we know that no other projects are using this.

**Review by aserv92 — CHANGES_REQUESTED** [review]
Please add some tests, and write a description for this PR.

**Review by aserv92 — CHANGES_REQUESTED** [review]
.

**Review by izazueta — CHANGES_REQUESTED** [review]
What Andrew said

**Review by izazueta — CHANGES_REQUESTED** [review]
.

File: src/Medology/Behat/StoreContext.php (hunk 1 above)
Comment (BKorenek) [reply src/Medology/Behat/StoreContext.php:41]: Remove what? Testing is already in place for this feature. This is simply adding in additional formatting options.

Comment (BKorenek) [reply src/Medology/Behat/StoreContext.php:41]: just realized you are looking at the one specific line. Yeah, that was added in by me and is not in use anywhere

**Review by PayneWheat — APPROVED** [review]
. (just wanted to fit in)

**Review by PayneWheat — APPROVED** [review]
.. (pull approve is buggy)

**Review by izazueta — APPROVED** [review]

**Review by aserv92 — APPROVED** [review]

**Pull request merged**